tests/scripts/components-basic-checks.sh

```diff
@@ -117,3 +117,10 @@ component_tf_psa_crypto_check_test_helpers () {
     msg "unit test: translate_ciphers.py"
     python3 -m unittest framework/scripts/translate_ciphers.py 2>&1
 }
+
+component_tf_psa_crypto_check_names () {
+    msg "Check: declared and exported names (builds the library)" # < 3s
+    # Make generated files required for check
+    cmake -DGEN_FILES=ON .
```

Comment:
Currently an error with this line, I believe this locks the Makefile resulting in a faliure down the line.

Reply:
Yes I understand this issue. Thinking about it ...

Reply:
To unblock things, I propose for the time being to do just:
```suggestion
    ./scripts/generate_driver_wrappers.py
```
This seems to work and we can revisit that when we thing more generally about the generated files in the context of TF-PSA-Crypto.